modules/custom/az_migration/migrations/migrate_plus.migration.az_media.yml

```diff
@@ -0,0 +1,52 @@
+id: az_media
+label: All Media (media_migration)
+migration_group: az_migration
+audit: true
+migration_tags:
+  - Drupal 7
+  - Content
+  - Quickstart Content Migration
+source:
+  plugin: d7_file_entity_item
+  constants:
+    bundle_prefix: az
+process:
+  uid: uid
```

Comment:
A question for PR review group discussion: this process pattern of `uid:uid` is enforcing that the UIDs of migrated users (and other types of entities where this pattern is used) are using the same id they were on the original site. This can sometimes cause content to get overwritten if they shared the same id as on the previous site. Is that behavior what we want to enforce?

For example, if the site already has any existing users, they will likely be overwritten by this migration.

The alternative is to not specifically list an ID mapping, in which case migrate will choose a new free id. It's still possible to know that ID later (through `migration_lookup` result). This also allows running the migration later if content was missed, or if it's needed to migrate two sites content to a single site.